rcli/call.py

```diff
@@ -53,25 +53,25 @@ def call(func, args):
     """
     assert hasattr(func, '__call__'), 'Cannot call func: {}'.format(
         func.__name__)
-    is_func = isinstance(func, FunctionType)
-    raw_func = func if is_func else func.__class__.__call__
+    raw_func = (
+        func if isinstance(func, FunctionType) else func.__class__.__call__)
     hints = collections.defaultdict(lambda: Any, get_type_hints(raw_func))
-    params, vararg, kwarg = get_signature(raw_func)
-    params += [vararg, kwarg]
+    signature = getargspec(raw_func)
+    params = list(signature[0]) + [s for s in signature[1:3] if s]
     keyword_args = {}
     positional_args = ()
     for k, nk, v in _normalize(args):
-        if nk == vararg:
+        if nk == signature[1]:
```

Comment:
Since `signature` is an `inspect.FullArgSpec` index it by name to (significantly) enhance readability.

e.g.

`if nk == signature.varargs`